requirements/requirements.txt

```diff
@@ -4,6 +4,7 @@ gym>=0.22.0
 jax>=0.2.26
 jaxlib>=0.1.74
 matplotlib>=3.3.4
+networkx==2.8.3
```

Comment:
While I understand `networkx` may come handy for graph rendering, we are trying to keep Jumanji as light as possible. Is there any way to implement rendering using Matplotlib instead? `GraphColoring` has rendering implemented using Matplotlib and most features should be transferable to CMST. Could you check with [Elsh](ElshadaiK)?

Reply:
I have removed the need for networkx. I wasn't using it for something important.